wunderio/next4drupal-project
# Nex 72: Social share component

[*Nex 72*](https://wunder.atlassian.net/browse/NEX-72)

*Changes proposed in this PR:*
- Create a social media share component

*How to test:*
1. Open the link and click on the share buttons to observe that they lead you to login into your social media account and a post should be available too.

2. Please note that there is an EmberJs console error for LinkedIn share link. Currently looking into it and contacting them. They have a js script plugin but would prefer not to use that.

*Link to feature environment:*
[Link](https://feature-nex-72-socialshare-next.next4drupal-project.dev.wdr.io/)
